doc/examples/applications/plot_human_mitosis.py

```diff
@@ -0,0 +1,170 @@
+"""
+================================
+Segment human cells (in mitosis)
+================================
+
+In this example, we analyze a microscopy image of human cells. We use data
+provided by Jason Moffat [1]_ through
+[CellProfiler](https://cellprofiler.org/examples/#human-cells).
+
+.. [1] Moffat J, Grueneberg DA, Yang X, Kim SY, Kloepfer AM, Hinkle G, Piqani B, Eisenhaure TM, Luo B, Grenier JK, Carpenter AE, Foo SY, Stewart SA, Stockwell BR, Hacohen N, Hahn WC, Lander ES, Sabatini DM, Root DE (2006) "A lentiviral RNAi library for human and mouse genes applied to an arrayed viral high-content screen" Cell, 124(6):1283-98. DOI: [10.1016/j.cell.2006.01.040](https://doi.org/10.1016/j.cell.2006.01.040). PMID: 16564017
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import ndimage as ndi
+
+from skimage import feature, filters, io, morphology, util
+
+
+image = io.imread('https://github.com/CellProfiler/examples/blob/master/ExampleHuman/images/AS_09125_050116030001_D03f00d0.tif?raw=true')
```

Comment:
To be safe, I would suggest to host our own copy. Any comment on this @scikit-image/core ?

Reply:
I agree but I would host it in gitlab.com/scikit-image, or a new github repo, rather than here.

Reply:
@sciunto thanks for reviewing! In the PR description, I wrote: "At the moment, I'm fetching the example data from a public GitHub repo; I'm happy to discuss whether we should include the image in the (newly available) pooch registry." <-- Do you have an opinion on the latter?

Reply:
This was my thought, ie include it in pooch. @emmanuelle Do you agree?

Reply:
ok, let's host in https://gitlab.com/scikit-image/data as suggested by @jni. That will be our first image in this repo :-). When you do this @mkcor please add the origin of this image in the Readme of the repo. Then for adding the image to the registry you can copy what's done for the `cells.tif` dataset.

Reply:
Ok, great! PR submitted over at: https://gitlab.com/scikit-image/data/-/merge_requests/2